packages/rich-text-editor/src/RichTextEditor/components/ToggleIconButton/ToggleIconButton.scss

```diff
@@ -18,7 +18,8 @@
     outline: none;
   }
 
-  &:global(.focus-visible)::after {
+  &:global(.focus-visible)::after,
+  &:focus-visible:after {
```

Comment:
The `.focus-visible` class polyfill should cover us for all cases, unless you discovered something I'm not aware of?

Reply:
Oh yes, so this was a weird one. There's actually a case where when you toggle the button with keyboard it looses the .focus-visible class entirely despite still retaining focus on the button, native focus-visible seems to catch this but the polyfill doesn't